eng/SharedFramework.External.props

```diff
@@ -78,6 +78,10 @@
     <ExternalAspNetCoreAppReference Include="@(_TransitiveExternalAspNetCoreAppReference)" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <_CompilationOnlyReference Include="Microsoft.Bcl.AsyncInterfaces" />
```

Comment:
@natemcmaster Here is a fun one, should I put the same comment as below, but modified slightly because the package is irrelevant on `>netstandard2.0`?

Reply:
I don't know what this assembly is. Is it like a polyfill for IAsyncEnumerable on ns2.0?

Reply:
Yeah

Reply:
Then yes, it's probably ok the classify this the same as the other compilation-only refs.